### PR TITLE
[ExtensionKit] Add support for Xcode 14 beta 1 & 2

### PR DESCRIPTION
**Pull request by SotoiGhost**

### CODE REVIEW

**Review by mandel-macaque — CHANGES_REQUESTED** [review]
There are a few things that are missing for this to work. 

1. You need to update https://github.com/xamarin/xamarin-macios/blob/main/src/frameworks.sources since you need to process the code
2. You need to add the new library paths here: https://github.com/xamarin/xamarin-macios/blob/main/tools/common/Frameworks.cs

I think that is all since this is a mac framework.

File: tools/common/Frameworks.cs [hunk 1]
```diff
@@ -277,6 +277,7 @@ public Framework Find (string framework)
 					{ "ScreenCaptureKit", "ScreenCaptureKit", 12,3 },
 
 					{ "BackgroundAssets", "BackgroundAssets", 13,0},
+          { "ExtensionKit", "ExtensionKit", 13,0 },
```
Comment (rolfbjarne) [inline tools/common/Frameworks.cs:280]: Spaces -> tabs

**Review by rolfbjarne — CHANGES_REQUESTED** [review]
You also need to add an entry in this list:

https://github.com/xamarin/xamarin-macios/blob/b1cb5f62b7cde5181c41df750b1b59fa7f2f0762/src/Foundation/NSObject.mac.cs#L39-L107

**Comment by rolfbjarne** [comment]
Also make sure to update the PR to the latest beta

File: tools/common/Frameworks.cs (hunk 1 above)
Comment (SotoiGhost) [reply tools/common/Frameworks.cs:280]: Weird...

image

Comment (SotoiGhost) [reply tools/common/Frameworks.cs:280]: I did a rebase, I'm seeing this now, don't know what happened

**Review by rolfbjarne — APPROVED** [review]

**Comment by dalexsoto** [comment]
@SotoiGhost please look at the intro failures

**Review by mandel-macaque — APPROVED** [review]

**Pull request merged**